app/event-list/event-list.component.html

```diff
@@ -14,7 +14,8 @@
                 <StackLayout row="1" col="0" backgroundColor="#e0e0e0" class="list-group-item">
                     <Label class="list-group-item-heading" [text]="event.name"></Label>
                     <Label class="list-group-item-text fa" text="&#xf073;" [text]="'Start Date: July 22, 2018'"></Label>
-                    <Label class="list-group-item-text fa" text="&#xf073;" [text]="'End Date: July 28, 2018'"></Label>          
+                    <Label class="list-group-item-text fa" text="&#xf073;" [text]="'End Date: July 28, 2018'"></Label>    
```

Comment:
We shouldn't be hardcoding the start/end dates and the location. I'd recommend creating an interface and referencing that here. You can use static data for now. We will switch to using the real data once the APIs are hooked up.